docs/filters.rst

```diff
@@ -5,12 +5,33 @@ Filtering
 .. py:module:: web3.utils.filters
 
 
-The :meth:`web3.eth.Eth.filter` method can be used to setup filter for:
+The :meth:`web3.eth.Eth.filter` method can be used to setup filters for:
+
+* Pending Transactions: ``web3.eth.filter('pending')``
+
+* New Blocks ``web3.eth.filter('latest')``
 
-* Pending Transactions
-* New Blocks
 * Event Logs
 
+    Through the contract instance api:
+
+    .. code-block:: python
+
+        event_filter = myContract.eventFilter('eventName', {'filter': {'arg1':10}})
```

Comment:
https://github.com/ethereum/web3.py/issues/642 indicates we could use an example which shows how to do filters that have string values as arguments, including a note about what types they can use for those arguments (do we require bytes?  can they use hex encoded values?)